permissions/CanAccess.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhubContrib\modules\jitsiMeet\permissions;
+
+use humhub\libs\BasePermission;
+use humhub\modules\user\models\Group;
+use Yii;
+
+class CanAccess extends BasePermission
+{
+
+    /**
+     * @inheritdoc
+     */
+    public $defaultAllowedGroups = [];

```

Comment:
Let the default state Allow for all groups.

Reply:
@luke- Done in commit https://github.com/humhub-contrib/jitsi-meet/pull/36/commits/12689e015aba8af63241b4a38c97b80e51cba257.